homeassistant/components/knx/device_trigger.py

```diff
@@ -63,17 +72,24 @@ async def async_get_trigger_capabilities(
         selector.SelectOptionDict(value=ga.address, label=f"{ga.address} - {ga.name}")
         for ga in project.group_addresses.values()
     ]
+
     return {
         "extra_fields": vol.Schema(
             {
-                vol.Optional(EXTRA_FIELD_DESTINATION): selector.SelectSelector(
+                vol.Optional(CONF_KNX_DESTINATION): selector.SelectSelector(
                     selector.SelectSelectorConfig(
                         mode=selector.SelectSelectorMode.DROPDOWN,
                         multiple=True,
                         custom_value=True,
                         options=options,
                     ),
                 ),
+                # these shall have matching default values as in TELEGRAM_TRIGGER_SCHEMA
+                vol.Optional(CONF_KNX_GROUP_VALUE_WRITE, default=True): cv.boolean,
+                vol.Optional(CONF_KNX_GROUP_VALUE_RESPONSE, default=True): cv.boolean,
+                vol.Optional(CONF_KNX_GROUP_VALUE_READ, default=True): cv.boolean,
+                vol.Optional(CONF_KNX_INCOMING, default=True): cv.boolean,
+                vol.Optional(CONF_KNX_OUTGOING, default=True): cv.boolean,
```

Comment:
Instead of this, can't trigger.py export something like:
```py
TELEGRAM_TRIGGER_OPTIONS = {
    vol.Optional(CONF_KNX_GROUP_VALUE_WRITE, default=True): cv.boolean,
    vol.Optional(CONF_KNX_GROUP_VALUE_RESPONSE, default=True): cv.boolean,
    vol.Optional(CONF_KNX_GROUP_VALUE_READ, default=True): cv.boolean,
    vol.Optional(CONF_KNX_INCOMING, default=True): cv.boolean,
    vol.Optional(CONF_KNX_OUTGOING, default=True): cv.boolean,
}
```

Then we can include it with `**TELEGRAM_TRIGGER_OPTIONS` both here and in `TELEGRAM_TRIGGER_SCHEMA`

Reply:
This would be possible for now.

I was planning to maybe change the extra_fields to selectors to be able to add descriptions once https://github.com/home-assistant/frontend/pull/20567 is finished. I think selectors would not be valid for the `trigger.py` schema, would they?

Reply:
I guess not, but we should not worry about things which may happen in the future now IMHO.